### ISSUE TITLE
Proposal: use 4 space indentation for perl and bash, removing tabs from files

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#1500]
We seem to have a conflict with `vim` and `emacs` comment setting headers in `perl` and `bash` files, the various indentation levels used in the files (4 vs 8), and whether we indent with tabs vs spaces.

Let's do the following:

1. Agree on 4 space indentation for `bash` and `perl` code
1. Remove all tabs from `bash` and `perl` code used as indentation replacing with 4 spaces
1. Update/add (where appropriate) the `vim` and `emacs` headers to specify 4 space indentation without the use of tabs

Can folks weigh in on this proposal?

**Comment by k-rister** [comment]
> 1. Agree on 4 space indentation for `bash` and `perl` code

Seems reasonable.

> 2. Remove all tabs from `bash` and `perl` code used as indentation replacing with 4 spaces

How are you going to accomplish this?  Run a `sed` on the whole repo?

> 3. Update/add (where appropriate) the `vim` and `emacs` headers to specify 4 space indentation without the use of tabs
 
Seems reasonable.

**Comment by portante** [comment]
> How are you going to accomplish this? Run a `sed` on the whole repo?

Yes, was thinking of some automated script that we could use to do this, with a commit made with only those changes so that we can review the automated script instead of inspecting every space/tab change.
